HGCalAnalysis/plugins/HGCalAnalysis.cc

```diff
@@ -359,22 +347,33 @@ HGCalAnalysis::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 	int reachedEE=0; // compute the extrapolations for the particles reaching EE and for the gen particles
 	if (myTrack.noEndVertex() || myTrack.genpartIndex()>=0)
 	  {
-	    // went up to calorimeter: propagate it
-	    reachedEE=1;
+
 	    RawParticle part(myTrack.momentum(),myTrack.vertex().position());
 	    part.setID(myTrack.id());
-	    BaseParticlePropagator myPropag(part,140,layerPositions[0],3.8);
+	    BaseParticlePropagator myPropag(part,160,layerPositions[0],3.8);
```

Comment:
What's the correct value to be used for the radius - 140 or 160? Does the choice of either of them have an effect here?